run_robot_test.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-set -eux
+set -x
```

Comment:
Is this intentional change? IMHO "-eu" wont harm anything

Reply:
Yes, they both had to be removed due to requirement changes
-e was removed because it causes the shell script to exit when there is robot test failure, as the exit status of robot command is 1
So if a test case fails, the bash script will exit right away and won't execute the code for re-run failed tests
-u was removed as it requires that every variable used should already be set, and we are using a Jenkins Environment Variable which outside of Jenkins run will be empty/un-set